REPLesent.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+
+REPLesent=$PWD/replesent.scala
+MSG="[REPLesent] -"
+
+if [ $# -eq 0 ]
+  then
+    echo "$MSG You need to supply the presentation file name as an argument!"
+    echo ""
+    echo "usage: $0 [yourpresentation.txt]"
+    exit 1;
+fi
+
+# Check that input is not absolute or relative path
+if [[ $1 == *"/"* ]]
+    then
+        echo "$MSG You can't use a relative or absolute path to locate the presentation!"
+        echo "$MSG Move first to the presentation directory and then call REPLesent from there."
+        exit 1;
+fi
```

Comment:
REPLesent.scala can't find files outside of its folder. We should properly handle the input parameter of the constructor:
```
case class REPLesent(
width: Int = 0
, height: Int = 0
, input: String = "REPLesent.txt" 
, slideCounter: Boolean = false
, slideTotal: Boolean = false
, intp: scala.tools.nsc.interpreter.IMain = null
) {
```
Try to reproduce it and let me know